flink/Dockerfile

```diff
@@ -0,0 +1,41 @@
+###############################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
@bryteise do we need this license notice here?

Reply:
What is the license for? The Dockerfile? That seems odd. I don't see why having a license there makes sense.

Reply:
I referred to https://github.com/docker-flink/docker-flink/blob/f96f07a516dbd4d9544d22c5272006ccd0b988d6/1.7/hadoop24-scala_2.11-debian/Dockerfile which have this license. If we do not need it , I will modify it. Thanks for your comments.